webpack/assets/javascripts/react_app/components/common/CardTemplate/CardItem.js

```diff
@@ -0,0 +1,102 @@
+import PropTypes from 'prop-types';
+import React, { useState } from 'react';
+import Skeleton from 'react-loading-skeleton';
+import {
+  DataList,
+  DataListItem,
+  DataListItemRow,
+  DataListItemCells,
+  DataListCell,
+  Card,
+  CardTitle,
+  CardBody,
+  Accordion,
+  AccordionItem,
+  AccordionContent,
+  AccordionToggle,
+  GridItem,
+} from '@patternfly/react-core';
+
+const CardItem = ({ content, header, children }) => {
```

Comment:
This seems to have pretty specific implementation details, do we want to have this for all Cards?
Shouldn't we just start with a `HostDetails/template/CardItem` and only once we tweek it and find another usage, let's generalize? Generalizing soon is usually bad idea.